packages/internals/src/get-generators/utils/check-feature-flags/checkFeatureFlags.ts

```diff
@@ -1,26 +1,12 @@
 import type { ConfigMetaFormat } from '../../../engine-commands'
 import { GetGeneratorOptions } from '../../getGenerators'
-import { forbiddenPreviewFeatureWithDataProxyFlagMessage } from './forbiddenPreviewFeatureWithProxyFlagMessage'
 
 /**
  * Check feature flags and preview features
  * @param config
  * @param options
  */
-export function checkFeatureFlags(config: ConfigMetaFormat, options: GetGeneratorOptions) {
-  checkForbiddenFeaturesWithDataProxyFlag(config, options)
-}
-
-function checkForbiddenFeaturesWithDataProxyFlag(config: ConfigMetaFormat, options: GetGeneratorOptions) {
-  options.dataProxy === true &&
-    config.generators.some((generatorConfig) => {
-      return generatorConfig.previewFeatures.some((feature) => {
-        if (feature.toLocaleLowerCase() === 'metrics'.toLocaleLowerCase()) {
-          throw new Error(forbiddenPreviewFeatureWithDataProxyFlagMessage('metrics'))
-        }
-      })
-    })
-}
+export function checkFeatureFlags(_config: ConfigMetaFormat, _options: GetGeneratorOptions) {}
```

Comment:
Why do we need to keep `checkFeatureFlags`?

Reply:
No really but thought we could leave it there for future. I can remove if you want.

Reply:
It's ok for me to keep like this 👍🏼